pkger/service.go

```diff
@@ -346,11 +346,12 @@ func (s *Service) resourceCloneToResource(ctx context.Context, r ResourceToClone
 		}
 		var cellViews []cellView
```

Comment:
you don't need cellViews anymore, just axe them and pass the dashboard as a whole to the dashboardToResource. Axe the type as well